<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>461</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>758</width>
    <height>461</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="cb_generate">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Generate saved state after simulation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="cb_postprocess">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Post-processing in Lizard</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QWidget" name="postprocessing_widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="cb_basec_results">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Basic processed results</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="basic_processed_widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="cb_arrival_time_map">
             <property name="text">
              <string>Arrival time map</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_damage_estimation">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Damage estimation</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="damage_estimation_widget" native="true">
          <layout class="QFormLayout" name="formLayout_3">
           <property name="leftMargin">
            <number>30</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Cost type</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Flood month</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="dd_flood_month"/>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="dd_cost_type"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Inundation period</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Repair time infrastructure</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Repair time buildings</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sb_repair_infrastructure">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>240</number>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="sb_repair_building">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>240</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="sb_period">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>397</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
     <property name="margin">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0" alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="pb_next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="cb_boundary">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Include boundary conditions</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="cb_conditions">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Include initial conditions</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QWidget" name="load_state_widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="cb_load_saved_state">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Load from saved state</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="cb_laterals">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Include laterals</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="cb_breanches">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Include breaches</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="cb_precipitation">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Include precipitation</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="cb_multiple_simulations">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Multiple simulations</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QWidget" name="multiple_simulations_widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>55</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>I want</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="dd_number_of_simulation"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>simulations</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Every simmulation has different </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="dd_simulation_difference"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
